Editor/Scripts/interactivetutorials_dialog.py

```diff
@@ -115,8 +121,10 @@ def __init__(self, parent=None):
         self.title_label.setProperty("class", "Headline")
         self.tutorial_layout.addWidget(self.title_label)
 
-        self.content_area = QTextEdit(self)
-        self.content_area.setReadOnly(True)
+        #self.content_area = QTextEdit(self)
+        #self.content_area.setReadOnly(True)
+        self.content_area = QLabel(self)
+        self.content_area.setWordWrap(True)
```

Comment:
I like this change! You can go ahead and just delete the old lines instead of commenting them out.